addons/mail/static/src/js/chat_window.js

```diff
@@ -140,6 +141,12 @@ return Widget.extend({
             this.trigger("fold_channel", this.channel_id, this.folded);
         }
     },
+    _onChatWindowClicked: function() {
+        var selectObj = window.getSelection();
+        if (selectObj.toString().length === 0){
```

Comment:
here, you should replace this line with: 
        if (selectObj.anchorOffset === selectObj.focusOffset){